yang/frr-vrf.yang

```diff
@@ -36,29 +40,32 @@ module frr-vrf {
           "VRF name.";
       }
 
-      leaf id {
-        type uint32 {
-          range "0..4294967295";
+      container netns {
```

Comment:
The VRF netns can be configured on zebra only, so this container probably belongs to frr-zebra.yang (which should augment `/frr-vrf:lib/vrf`).

Reply:
Sure will exclude this from here and submit in a new PR.